EmoPy/examples/fermodel_example_webcam.py

```diff
@@ -0,0 +1,41 @@
+#This file allows to perform Emotion detection on frames grabbed from the webcam using OpenCV-Python
+
+import cv2
+import sys
+sys.path.append('../')
+
+#Choose the type of Face Expression Model
+from src.fermodel import FERModel
+
+#Frame Number
+FRAME_NUM = 0
+
+#Choose the type of face detector cascade you want to use
+cascPath = "~/EmoPy/venv/lib/python3.5/site-packages/cv2/data/haarcascade_frontalface_default.xml"
+faceCascade = cv2.CascadeClassifier(cascPath)
+#Specify the camera which you want to use. The default argument is '0'
+video_capture = cv2.VideoCapture(0)
+
+while True:
+	#Capture frame-by-frame
+	ret, frame = video_capture.read()
+	#Save the captured frame on disk
+	file = '~/EmoPy/models/examples/image_data/image.jpg'
```

Comment:
My system wasn't able to access this file when run locally. Similar to my other comment, using relative file paths such that any machine can find them will work best 👍